sample-apps/00-fluentBit/kubernetes/fluentbit.yaml

```diff
@@ -91,8 +93,18 @@ data:
         Reserve_Data      On
         Preserve_Key      On
 
+    [FILTER]
+        Name                     grep
+        Match                    *
+        Regex                    traceId [^ ]+
+    
+    [FILTER]
+        Name    grep
+        Match   *
+        Exclude traceId 0
```

Comment:
Not all log calls are generated with the traceid value because this has to go through a library that performs this instrumentation. For example, in the application startup, only after a certain moment that the lib starts to be included do we have this metadata added. So I don't think we should remove everything that doesn't have a traceid but I would like to hear your views.

Reply:
Agree with that, on this issue, I noticed that some logs don't have traceID and some logs have traceID as 0. Could you help to advice the case when traceID is 0. What does those traceID mean?